sdk/anomalydetector/azure-ai-anomalydetector/src/samples/java/com/azure/ai/anomalydetector/DetectAnomaliesLastPoint.java

```diff
@@ -56,9 +57,10 @@ public static void main(final String[] args) throws IOException {
         InputStream fileInputStream = new FileInputStream("azure-ai-anomalydetector\\src\\samples\\java\\sample_data\\request-data.json");
         JsonReader reader = Json.createReader(fileInputStream);
         BinaryData detectBody = BinaryData.fromString(reader.readObject().toString());
-        RequestOptions requestOptions = new RequestOptions();
-        Response<BinaryData> response = anomalyDetectorClient.detectUnivariateLastPointWithResponse(detectBody, requestOptions);
-        System.out.println(response.getValue().toString());
 
+        DetectRequest detectRequest = detectBody.toObject(DetectRequest.class);
+        LastDetectResponse lastDetectResponse = anomalyDetectorClient.detectUnivariateLastPoint(detectRequest);
```

Comment:
If your sample recommend user to load the JSON, then call the API, why do you need model in the first place...

Reply:
Yes. That is a question. In fact, I think Json is better to work with, in comparison with setting object attributes one by one (I tried but gave up.).

Reply:
Update all.